db/migrate/20210218135141_update_postal_code_registration.rb

```diff
@@ -3,18 +3,23 @@
 class UpdatePostalCodeRegistration < ActiveRecord::Migration[5.2]
   def up
     affected_users = users_with_postal_code
-    updated_users = 0
 
-    affected_users&.each do |user|
-      next unless user[:address].present? && user[:address]['postal_code'].present?
+    unless affected_users.present? && affected_users.respond_to?(:each)
+      puts 'No users affected by migration, migration aborted !'
+      return
+    end
```

Comment:
```suggestion
      return if affected_users.empty
```
No need to puts empty values. Empty is sufficient.